docs/5g-on-helium/mining-5g.mdx

```diff
@@ -0,0 +1,169 @@
+
+:::
+
+## How Do Hotspots Earn Helium MOBILE Tokens?
+
+The Helium blockchain rewards 5G Hotspots for providing 5G wireless coverage and verifying the
+Helium Network. 5G Hotspots are rewarded in the Helium MOBILE Token.
+
+During the Genesis Period, 5G Hotspots will earn MOBILE by reporting their status via a “heartbeat”,
+an automatic process of reporting to the Blockchain that the Small Cell Radio is online and
+registered with the SAS.
+
+Once your 5G Hotspot is completely deployed and fully synced with the Helium blockchain, you as the
+owner are not required to do anything else in order to earn MOBILE. Your 5G Hotspot will perform all
+of its activities on its own and will be rewarded for maintaining a valid connection to the SAS.
+
+## Emissions Curve
+
+There will be a max supply of 250B MOBILE with issuance halvenings of every 2 years aligned with the
+HNT issuance halvenings. The first halving will occur on August 1, 2023 to align with HNT halving
+and will continue on a 2 year cycle afterward.
+
+This allocation is distributed throughout the shortened 1 year period to online 5G Hotspots on an
+epochal basis, weighted by Small Cell model.
+
+For clarity, the emission schedule is as follows:
+
+|  Phase  | Weekly Emission |
+| :-----: | --------------: |
+| Genesis |     640,000,000 |
+| 5G PoC  |   1,280,000,000 |
+
+The Helium Foundation will work with the core developers to begin the Genesis period of MOBILE
+rewards. This will be paid out of the Foundation's pre-mined MOBILE rewards pool. The rate of
+rewards, as defined above, is half that defined in HIP 53 and will be proportionally distributed to
+all active 5G radios as described below.
+
+## Small Cell Type Reward Scaling
+
+All properly installed and registered 5G Hotspots will earn rewards for participating on the
+Network. A rewards unit multiple is applied to the various Small Cell Models based on whether they
+are indoor or outdoor, as well as their power output, as follows:
+
+| Model           | Reward Weight | Small Cell Type    |
+| :-------------- | :-----------: | :----------------- |
+| Nova 436H       |     2.0x      | High Power Outdoor |
+| Nova 430i       |     1.5x      | Outdoor            |
+| Sercomm Outdoor |     1.5x      | Outdoor            |
+| Sercomm Indoor  |     1.0x      | Indoor             |
+| Neutrino 430    |     1.0x      | Indoor             |
+
+Individual rewards per period are calculated as a proporation of the total weighted reward units.
+
+## Earnings Example
+
+Earnings are calculated in 3 steps
+
+1. Calculate the Total Emissions Weight and Emissions Weight Ratio for each Small Cell Type
+2. Each Small Cell Type is allocated its ratio of the total rewards in the given epoch
+3. Each 5G Hotspot is rewarded from its respective rewards pool by Small Cell Type
+
+As an example, we can calculate the rewards for a 5G Hotspot in a single epoch assuming the
+following:
+
+- 100,000 MOBILE Rewarded per epoch
```

Comment:
For number of Mobile rewards, I would have a number that is close to what actual emission will be per epoch.